ui/src/app/job-details/tasks/tasks.component.html

```diff
@@ -49,20 +51,27 @@
         <mat-header-cell *matHeaderCellDef> Attempts </mat-header-cell>
         <mat-cell *matCellDef="let t" class="column">{{ t.attempts }}</mat-cell>
       </ng-container>
-      <ng-container matColumnDef="stdout">
-        <mat-header-cell *matHeaderCellDef> Stdout </mat-header-cell>
+      <ng-container matColumnDef="logs">
+        <mat-header-cell *matHeaderCellDef> Log files </mat-header-cell>
         <mat-cell *matCellDef="let t" class="column">
-          <a class="log-item" href="{{ getResourceUrl(t.stdout) }}">
-            {{ getResourceFileName(t.stdout) }}
-          </a>
-        </mat-cell>
-      </ng-container>
-      <ng-container matColumnDef="stderr">
-        <mat-header-cell *matHeaderCellDef> Stderr </mat-header-cell>
-        <mat-cell *matCellDef="let t" class="column">
-          <a class="log-item" href="{{ getResourceUrl(t.stderr) }}">
-            {{ getResourceFileName(t.stderr) }}
-          </a>
+          <clr-tooltip *ngIf="getResourceUrl(t.stdout)">
+            <a class="log-item" href="{{ getResourceUrl(t.stdout) }}">
```

Comment:
So the link goes into the tooltip? Should the icon also be a link? Could you provide a link to the mocks for this? Couldn't find this interaction.

Reply:
Per offline discussion, apparently this does put the link onto the icon and not in the tooltip. I find the structure of this HTML surprising, but it appears to be the intended usage.